components/Dashboard/candidate/profile/add-education-modal.tsx

```diff
@@ -0,0 +1,130 @@
+import React,{useEffect, useState} from 'react'
+import Modal from '@/components/modal/Modal'
+import { CancelIcon } from '@/assets';
+import { DateInput, Input } from '@/components/Input';
+
+import { Button, OutlineBtn } from '@/components/button';
+import PrimarySelect from './primary-selects';
+interface modalcompProps {
+    open: boolean;
+    setOpen: any;
+    onClick: any;
+    loading: boolean;
+    setData: any;
+}
+
+
+
+const fieldsOfStudy = [
+    "Computer Science",
+    "Engineering",
+    "Mathematics",
+    "Physics",
+    "Biology",
+    "Chemistry",
+    "Psychology",
+    "Sociology",
+    "Business Administration",
+    // Add more fields as needed
+];
+const AddEducationModal: React.FC<modalcompProps> = ({ open, setOpen, onClick, setData, loading }) => {
+    const [name, setName] = useState("");
+    const [degreeType, setDegreeType] = useState("");
+    const [startDate, setStartDate] = useState("");
+    const [endDate, setEndDate] = useState("");
+    const [desc, setDesc] = useState("");
+    const [field, setField] = useState<any>({});
+    const fieldsOfStudyList = fieldsOfStudy.map(field => ({
+        name: field.toLowerCase().replace(/\s+/g, "_"),
+        label: field,
+    }));
+
+    useEffect(() => {
+        const bodyData = [
+            {
+                "institution_name": name,
+                "degree": degreeType,
+                "field_of_study": field?.name,
+                "date_start": startDate,
+                "date_end": endDate,
+                "description": desc
+            }
+        ];
+
+        setData(bodyData);
+
+    }, [name,degreeType,startDate,endDate,desc,field])
+    
+
+
+  return (
+    <div className='relative'>
+          <Modal open={open} setOpen={setOpen} >
+              <div className="w-full flex flex-col ">
+                  <div className="flex justify-end">
+                      <span onClick={() => setOpen(false)}>
+                          <CancelIcon />
+                      </span>
+                  </div>
+                  <div className="lg:w-10/12 w-full mx-auto mt-4 flex flex-col">
+                      <h1 className='text-black text-2xl lg:text-4xl font-bold'>
+                          Update Educational Qualification
```

Comment:
change the text to 'Add Educational Qualification', since the button that triggers this modal says 'add education' and not 'edit education'